ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ZAAppDelegateConnectorChromecastProtocol.h
+//  ZappAppConnector
+//
+//  Created by Avi Levin on 21/12/2017.
+//  Copyright © 2017 Applicaster Ltd. All rights reserved.
+//
+@protocol ZAAppDelegateConnectorChromecastProtocol
```

Comment:
Please add a space after `@protocol ...` and before `@end` to make it cleaner.

Reply:
You mean "enter" not "space". done :)